ntbs-service/DataMigration/NotificationImportService.cs

```diff
@@ -250,5 +255,57 @@ private async Task<List<string>> FilterOutImportedIdsAsync(PerformContext contex
             }
             return legacyIdsToImport;
         }
+        
+        private async Task MigrateOrUpdateCaseManager(Notification notification,
+            PerformContext context,
+            string requestId)
+        {
+            var legacyNotificationCaseManager =
+                (await _migrationRepository.GetNotificationsById(new List<string> {notification.LegacyId}))
+                .Select(n => n.CaseManager)
+                .SingleOrDefault();
+            if (legacyNotificationCaseManager == null)
+            {
+                return;
+            }
+            var existingCaseManager = await _userRepository.GetUserByUsername(legacyNotificationCaseManager);
+            var ntbsNotificationCaseManager = new User();
+            if (existingCaseManager != null)
+            {
+                ntbsNotificationCaseManager = existingCaseManager;
+            }
+            else
+            {
+                ntbsNotificationCaseManager.IsActive = false;
+                ntbsNotificationCaseManager.Username = legacyNotificationCaseManager;
+            }
```

Comment:
This could be rewritten as something like
```
var ntbsNotificationCaseManager = existingCaseManager
    ?? new User { IsActive = false, Username = legacyNotificationCaseManager };
```